src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Convert to the new synthetic runtimes
```

Comment:
Convert what to the new synthetic runtimes?

How about something like "Start using the new synthetic monitoring runtimes"?

Reply:
(Also, I think in general we want to be consistent using "synthetic monitoring" instead of just "synthetic" by itself.)